### DISCUSSION TITLE
About AFK symbol customisation

### DISCUSSION THREAD

**Discussion by ryanho720** [Zrips/CMI#5771] [Q&A]
Hi there, the following paragraph is non-existent in my Locale_EN.yml file. Where do I find it? Thanks a lot
This is my Locale_EN.yml file btw: https://pastebin.com/ziuEvRnh

```
    vanishSymbolOn: '&8[&7H&8]&r'
    vanishSymbolOff: ''
    afkSymbolOn: '&8[&7Afk&8]&r'
    afkSymbolOff: ''
```

**Comment by CloudeLecaw** [comment]
You find then in cmilib/Translation/language.yml folder.
